integration/tests/chat/channel-settings.spec.ts

```diff
@@ -0,0 +1,91 @@
+import { expect, test } from '@playwright/test'
+import path from 'path'
+import { CHANNEL_BANNERS } from '../../../common/flags'
+import { ChatPage } from '../../pages/chat-page'
+import { LoginPage } from '../../pages/login-page'
+
+let loginPage: LoginPage
+let chatPage: ChatPage
+
+test.beforeEach(async ({ page }) => {
+  loginPage = new LoginPage(page)
+  chatPage = new ChatPage(page)
+})
+
+test('changing channel banner', async ({ page }) => {
```

Comment:
Part of the point of these tests is that they should also test what happens when an innappropriate image is uploaded, so you need a test that does that too. You'll need to flesh out the fake google API to do this, probably just hash the image or check its filesize or something to decide whether it's inappropriate, don't think it really matters too much.